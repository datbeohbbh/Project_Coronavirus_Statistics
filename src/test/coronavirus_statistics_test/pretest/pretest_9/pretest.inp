7 16
? 5
? 5
? 5
q 1 5 36
? 7
q 1 7 5
? 4 1
q 3 7 1
q 2 5 21
? 4 1
q 3 7 1
? 4 1
q 3 5 15
? 7
? 4 1
? 4 1