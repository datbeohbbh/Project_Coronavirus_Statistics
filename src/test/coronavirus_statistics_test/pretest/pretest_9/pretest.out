Query ? 5 :
The number of safe cities is 7

Query ? 5 :
The number of safe cities is 7

Query ? 5 :
The number of safe cities is 7

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 5
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 4
-> Has recovered : 0
-> Was dead : 1

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 2

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 2 people

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 2

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 2

